treeherder/etl/perf.py

```diff
@@ -67,7 +67,8 @@ def _get_signature_hash(signature_properties):
 
 
 def load_perf_artifacts(project_name, reference_data, job_data, datum):
-    perf_datum = json.loads(datum['blob'])
+    blob = json.loads(datum['blob'])
+    perf_datum = blob['performance_data']
```

Comment:
this spot is failing in the tests, do we always assume 'performance_data' exists in the blob?